src/App.js

```diff
@@ -1,26 +1,45 @@
 import React, { useState } from "react";
+import { BrowserRouter, Route, Switch } from "react-router-dom";
+import { Link } from "react-router-dom";
 
-import { Workout } from "./Components/Workout/Workout";
+import Paper from "@material-ui/core/Paper";
+import Tabs from "@material-ui/core/Tabs";
+import Tab from "@material-ui/core/Tab";
+import SearchIcon from "@material-ui/icons/Search";
+import CreateIcon from "@material-ui/icons/Create";
 
-import "./App.css";
+// view components
+import ExerciseFinder from "./Views/ExerciseFinder.jsx";
+import RecordWO from "./Views/RecordWO.jsx";
+
+// custom components
+
+// custom styles
+import { ThemeProvider } from "@material-ui/core";
+import theme from "./theme";
 
 const App = () => {
-  const [workouts, setWorkouts] = useState([
-    { name: "Squats", description: "Squats description example" },
-    { name: "Deadlifts", description: "Deadlifts description example" },
-    { name: "Lunges", description: "Lunges description example" },
-  ]);
+  const [value, setValue] = React.useState(0);
```

Comment:
Hey @jandony , here you used React.useState but we already import React and useState at the very top so here, you could just use : const [value, setValue] = useState(0);

Reply:
Good catch! I'll remove that next.